src/Nox.Types.Abstractions/Enums/NoxType.cs

```diff
@@ -10,22 +10,50 @@ public enum NoxType : uint
     Object = 2524865811,
 
     // Compound Types - requires multiple fields to persist
+
+    [CompoundType]
+    [CompoundComponent("Start", typeof(DateTime))]
```

Comment:
DatetimeOffset work well with Odata and RestApi, was there any reason to change this?

Reply:
There was an error mapping values using the OData context:

can't parse JSON.  Raw result:

{"@odata.context":"https://localhost:7024/api/$metadata#AllNoxTypes","value":[{"statusCode":500,"message":"Error occured during request: /api/AllNoxTypes.Error: Unable to cast object of type \u0027System.DateTime\u0027 to type \u0027System.DateTimeOffset\u0027."}

But with DateTime it works well. Changed it because of that.

![image](https://github.com/NoxOrg/Nox.Generator/assets/136477073/0f896315-cdff-491c-b58a-1a7f34746f9d)

Reply:
![image](https://github.com/NoxOrg/Nox.Generator/assets/136477073/44cc8866-d6cd-419a-a9da-c37f7da66eaa)

Reply:
this is related with todays discussion, we need to put it back to DatetimeOffset, and fix the DateTimeRange (in another PR), so comment all noxtype DatetimeTange for now